src/blueprints/tv2/value-objects/tv2-studio-blueprint-configuration.ts

```diff
@@ -1,87 +1,90 @@
-  GraphicIgnoreStatus: boolean
-  GraphicNetworkBasePath: string
+  cameraSources: Tv2SourceMappingWithSound[]
+  remoteSources: Tv2SourceMappingWithSound[]
+  replaySources: Tv2SourceMappingWithSound[]
+  studioMicrophones: string[]
+  mediaPlayers: Tv2MediaPlayer[]
+  videoMixerBasicConfiguration: Tv2VideoMixerBasicConfiguration
+  casparCgPreRollDuration: number
+  serverPostRollDuration: number
+  splitScreenFolder?: Tv2FolderConfiguration
+  jingleFolder?: Tv2FolderConfiguration
+  graphicsFolder: Tv2FolderConfiguration
+  selectedGraphicsType: Tv2GraphicsType
+  vizPilotGraphics: Tv2VizPilotGraphics
+  htmlGraphics?: Tv2HtmlGraphics
+  preventOverlayWhileFullscreenGraphicsIsOnAir: boolean
+}
+
+export interface Tv2FolderConfiguration {
+  name?: string
+  networkBasePath: string
+  fileExtension: string
+  mediaFlowId: string
+  ignoreMediaStatus: boolean
 }
 
 export enum Tv2GraphicsType {
   VIZ = 'VIZ',
   HTML = 'HTML'
 }
 
-export interface Tv2VizGraphics {
-  CleanFeedPrerollDuration?: number
-  KeepAliveDuration: number
-  PrerollDuration: number
-  OutTransitionDuration: number
-  CutToMediaPlayer: number
-  FullGraphicBackground: number
+export interface Tv2VizPilotGraphics {
+  msBeforeShowingBeforeShowingOnCleanFeed?: number
+  msKeepOldPartAliveBeforeTakingGraphics: number
+  msPreRollBeforeTakingPilotGraphics: number
+  msKeepPilotGraphicsAliveBeforeTakingNext: number
+  msFromStartBeforeCuttingToBackgroundSource: number
+  backgroundVideoSwitcherSource: number
 }
 
 export interface Tv2HtmlGraphics {
-  KeepAliveDuration: number
-  GraphicURL: string
-  TransitionSettings: {
+  msKeepOldPartAliveBeforeTakingGraphics: number
+  graphicsUrl: string
+  transitionSettings: {
     wipeRate: number,
     borderSoftness: number
   }
 }
 
 export interface Tv2SourceMapping {
-  _id: string
-  SourceName: string
-  SwitcherSource: number
+  id: string
+  name: string
+  videoMixerSource: number
 }
 
 export interface Tv2MediaPlayer extends Tv2SourceMapping { }
 
 export interface Tv2SourceMappingWithSound extends Tv2SourceMapping {
-  SisyfosLayers: string[]
-  StudioMics: boolean
-  WantsToPersistAudio?: boolean
-  AcceptPersistAudio?: boolean
+  sisyfosLayers: string[]
+  studioMicrophones: boolean
+  wantsToPersistAudio?: boolean
+  acceptPersistAudio?: boolean
 }
 
-export interface Tv2VideoMixerSources {
-  Default: number
-  SplitArtFill: number
-  SplitArtKey: number
-  DSK: Tv2DownstreamKeyer[]
-  Dip: number
+export interface Tv2VideoMixerBasicConfiguration {
+  defaultVideoMixerSource: number
+  splitScreenArtFillSource: number
+  splitScreenArtKeySource: number
+  downstreamKeyers: Tv2DownstreamKeyer[]
+  dipVideoMixerSource: number
 }
 
 export interface Tv2DownstreamKeyer {
-  _id: string
-  Number: number
-  Key: number
-  Fill: number
-  DefaultOn: boolean
-  Roles: Tv2DownstreamKeyerRole[]
-  Clip: number,
-  Gain: number
+  id: string // This id isn't really used
+  name: number // For some reason this is a number
```

Comment:
It being named `name` implies that it is of type string, but in this case it is a number. The data being mapped into this field comes from a column of cells where the content is literally `[1, 2, 3, ...]`. Keeping the name as number makes sense in my mind, of cource with proper capalization. This will also allow you to remove the comment.

Reply:
Core saves them as `[0, 1, 2, ...]` though, which is why one is added to location where it is used.

Reply:
It's no longer called `name`